api/client/stats_helpers.go

```diff
@@ -24,6 +24,7 @@ type containerStats struct {
 	NetworkTx        float64
 	BlockRead        float64
 	BlockWrite       float64
+	PidsCurrent      uint64
```

Comment:
Where is this being set? I think we need something like this: https://github.com/docker/docker/commit/103cc2369e8c5d6e7264552a7e7f7f990aba1417#diff-234033fe3b8730c35e53ce0d8dbda485R119